coral-component-steplist/src/styles/index.styl

```diff
@@ -41,3 +43,42 @@
     display: none;
   }
 }
+
+._coral-Steplist-item[disabled] {
+  ._coral-Steplist-label,
+  ._coral-Steplist-markerContainer {
+    // Don't shoe cursor when the item is disabled.
+    cursor: default;
+  }
+}
+
+._coral-Steplist-accessibilityLabel {
+  bottom: 2.2rem;
+  font-size: 1px;
+}
+
+._coral-Steplist-item,
+._coral-Steplist-link {
+  &:focus {
```

Comment:
Can you please add a screenshot of the focus ring ?

Reply:
![image](https://user-images.githubusercontent.com/154077/82463328-b28d7200-9a8a-11ea-8773-3d58b7988c0a.png)